pkg/cgroups/cgroups.go

```diff
@@ -0,0 +1,66 @@
+package cgroups
+
+import (
+	"fmt"
+	"os"
+	"path"
```

Comment:
nit because we're specifically on Linux here and the two are equivalent: file paths should be manipulated with `path/filepath` rather than `path`

Reply:
Done.